packages/instanceserver/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import { updateAppConfig } from '@xrengine/server-core/src/updateAppConfig'
 
+require('fix-esm').register()
```

Comment:
Lets move this to createApp or appconfig in server-core, to also patch it for servers generally, since there may be imports to the engine from there in the future, and it seems generally good to have